api/groups/transactionGroup.go

```diff
@@ -387,23 +401,28 @@ func (tg *transactionGroup) sendMultipleTransactions(c *gin.Context) {
 	var start time.Time
 	txsHashes := make(map[int]string)
 	for idx, receivedTx := range ftxs {
-		var innerTx *transaction.Transaction
-		if receivedTx.InnerTransaction != nil {
-			innerTx, _, err = tg.createTransaction(receivedTx.InnerTransaction, nil)
-			if err != nil {
-				c.JSON(
-					http.StatusBadRequest,
-					shared.GenericAPIResponse{
-						Data:  nil,
-						Error: fmt.Sprintf("%s: %s", errors.ErrTxGenerationFailed.Error(), err.Error()),
-						Code:  shared.ReturnCodeRequestError,
-					},
-				)
-				return
+		innerTxs := make([]*transaction.Transaction, 0, len(receivedTx.InnerTransactions))
+		if len(receivedTx.InnerTransactions) != 0 {
+			for _, innerTx := range receivedTx.InnerTransactions {
+				if len(innerTx.InnerTransactions) != 0 {
+					// if one of the inner txs is invalid, break the loop and move to the next transaction received
+					break
+				}
+
+				newInnerTx, _, err := tg.createTransaction(innerTx, nil)
+				if err != nil {
+					// if one of the inner txs is invalid, break the loop and move to the next transaction received
```

Comment:
why would you send the tx further if the inner tx is wrong? Why not error?

Reply:
right, updated